src/main/java/com/google/sps/data/AccessSecret.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+import java.io.IOException;
+
+public class AccessSecret {
+  private String key;
+
+  public AccessSecret() throws IOException{
+    this.key = accessSecretVersion();
+  }
+
+  public String accessSecretVersion() throws IOException {
+    String projectId = "team-38-step-2020";
+    String secretId = "API_KEY";
+    String versionId = "2";
+    return accessSecretVersion(projectId, secretId, versionId);
+  }
+
+  public String accessSecretVersion(String projectId, String secretId, String versionId)
```

Comment:
this looks like an internal only method. should it be private

Reply:
done